feature/4465-emissions-view-data-format

jwhitehead77 opened the pull request:
added logic to format data properly

Commit:
added logic to format data properly

fanwari1 commented on src/utils/selected-emission-view.ts:
```diff
@@ -53,13 +53,33 @@ export async function getSelectedView(
-  );
+  )).map((item) => {
+    const props = Object.entries(item).map(([key, val]) => {
+      if (val !== null && val != undefined) {
```
Should this be changed to !== instead of != ?

jwhitehead77 committed:
Update selected-emission-view.ts

changed line 63
from if (val !== null && val != undefined) {
to if (val !== null && val !== undefined) {

jwhitehead77 committed:
Merge branch 'develop' into feature/4465-emissions-view-data-format

fanwari1 reviewed (approved):
lgtm
